scripts/firepit-builder/pipeline.js

```diff
@@ -33,25 +33,25 @@ cd("firepit_pipeline");
 const workdir = pwd();
 
 npm("init", "-y");
-npm("install", firebaseToolsPackage);
+
+if (fs.existsSync(firebaseToolsPackage)) {
+  cd(firebaseToolsPackage);
+  npm("pack");
+  cd(workdir);
+  const packedModule = path.join(firebaseToolsPackage, "*.tgz");
+  npm("install", packedModule);
+  rm(packedModule);
+} else {
+  npm("install", firebaseToolsPackage);
+}
 
 const packageJson = JSON.parse(cat("node_modules/firebase-tools/package.json"));
 const releaseTag = `v${packageJson.version}`;
 echo(`Installed firebase-tools@${packageJson.version}, using tag ${releaseTag}`);
 
-if (isLocalFirepit) {
-  echo("Using local firepit for testing...");
-  mkdir("firepit");
-  rm("-rf", path.join(__dirname, "../vendor/node_modules"));
```

Comment:
Why dont you need to `rm` these files anymore?

Reply:
This whole step was only used when the `--local` flag was used which was only helpful when this script made a distinction between `local` (i.e. build from current directory) and `remote` (build from HEAD on Github). Since this script no longer support the remote flow and now does all work in a temp directory, we can simplify this and allow the cleaning of the temp dir to do this removal.

Reply:
Ok makes sense.  Can you add a comment to the top of this file explaining what flags are accepted?  Since it just examines `argv` at different times it's kind of hard to know.